awsConfig.ts

```diff
@@ -0,0 +1,19 @@
+export const awsConfig = {
+    accessKeyId: "AKIAIOSFODNN7EXAMPLE", // Placeholder, avoid hardcoding AWS credentials
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID || "",
+    secretAccessKey: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY", // Placeholder
+    region: "us-east-1"
+};
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Remove hardcoded AWS credentials</b></div>
<div id="fix">

Consider removing hardcoded AWS credentials and using environment variables consistently. Line 2 contains a hardcoded `accessKeyId` which is overwritten by environment variable in line 3, while line 4 has a hardcoded `secretAccessKey`.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
export const awsConfig = {
    accessKeyId: process.env.AWS_ACCESS_KEY_ID || "",
    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY || "",
    region: "us-east-1"
};
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/36#issuecomment-2551516876>#39790f</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged